src/parquet/file/file-serialize-test.cc

```diff
@@ -61,40 +61,55 @@ class TestSerialize : public ::testing::Test {
  protected:
   NodePtr node_;
   SchemaDescriptor schema_;
+
+  void FileSerializeTest(Compression::type codec_type) {
+    std::shared_ptr<InMemoryOutputStream> sink(new InMemoryOutputStream());
+    auto gnode = std::static_pointer_cast<GroupNode>(node_);
+    std::shared_ptr<WriterProperties> writer_properties =
+        std::make_shared<WriterProperties::Builder>()
```

Comment:
I think you can stack allocate to avoid the extra gymnastics, but no need to change